### ISSUE TITLE
OPENAI API key not recognized

### ISSUE THREAD

**Issue by javableu** [Significant-Gravitas/Auto-GPT#2892]
### ⚠️ Search for existing issues first ⚠️

- [X] I have searched the existing issues, and there is no existing issue for my problem

### Which Operating System are you using?

Windows

### Which version of Auto-GPT are you using?

Latest Release

### GPT-3 or GPT-4?

GPT-3.5

### Steps to reproduce 🕹

I just did the last pull and the openAI API is not recognized anymore

### Current behavior 😯

Please set your OpenAI API key in .env or as an environment variable.
You can get your key from https://platform.openai.com/account/api-keys

### Your prompt 📝

```yaml
# Paste your prompt here
```

### Your Logs 📒

```log
<insert your logs here>
```

**Comment by ntindle** [comment]
reset your .env file and try again

**Comment by javableu** [comment]
Hi, 

I deleted the env file and restarted again. Same bug. It happened with the last update pulled an hour ago. Maybe something to do with the overload? I'm getting a print (os.getenv("OPENAI_API_KEY")) >> NONE in the config.py.

**Comment by holisHsu** [comment]
Check your PYTHONPATH 🤔

**Comment by javableu** [comment]
Maybe had to do with the Python path. I checked on an Ubuntu virtual machine and everything is fine. On widows 11 it failed.I used conda...

**Comment by Rumpa112** [comment]
I have the same problem and I have tried the same suggestion already. Also this version of AutoGPT is sooo confused and repeats itself a trillion times =)

**Comment by ntindle** [comment]
This is a setup issue. We’d love to help everyone, but discord is better suited to do so. Join and shoot a message in the #tech-support channel so we can get this fixed

**Comment by Rumpa112** [comment]
I've learned not to argue reason or logic with the AI 😆 so I might just as
well try the same here and not use arguments like "how can this be a
"setup/configurations" issue when I have tried doing a complete reinstall
of autogpt and made sure im on the stable branch. And I still get this
error constantly. Instead I will ask you, what's the fix! 😘

Den sön 23 apr. 2023 04:27Nicholas Tindle ***@***.***> skrev:
